pkg/apis/vpc/v1alpha1/networkinfo_types.go

```diff
@@ -39,6 +39,9 @@ type VPCState struct {
 	LoadBalancerIPAddresses string `json:"loadBalancerIPAddresses,omitempty"`
 	// Private CIDRs used for the VPC.
 	PrivateIPs []string `json:"privateIPs,omitempty"`
+	// supportedNetworkingStack indicates the networking stack supported by the VPC.
+	// Valid values: FULL_STACK_VPC, VLAN_BACKED_VPC
```

Comment:
I suppose you can have a kubebuilder marker to limit the valid values to these two?

// +kubebuilder:validation:Enum=FULL_STACK_VPC;VLAN_BACKED_VPC

Reply:
Done